.github/workflows/deploy.yml

```
@@ -20,13 +19,17 @@ jobs:
+      run: |
+        cd docs # Navigate to the correct directory if `package.json` is in `docs`
+        yarn
 
     - name: Build site
-      run: yarn build
+      run: |
+        cd docs # Ensure we're in the correct directory
+        yarn build
 
     - name: Deploy to GitHub Pages
       uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.4)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)